File: B/PHP/Disk.php
<?php

class Disk{
    private $type = 0;
    private $capacity = 0;
    private $priceDisk = 0;

    public function __construct($type = 0, $capacity=0, $priceDisk = 0){
        $this->type = $type;
        $this->capacity = $capacity;
        $this->priceDisk = $priceDisk;
    }

    public function setType($t){
        $this->type = $t;
    }

    public function getType(){
        return $this->type;
    }

    public function setCapacity($t){
        $this->capacity = $t;
    }

    public function getCapacity(){
        return $this->capacity;
    }

    public function setPriceDisk($p){
        $this->priceDisk = $p;
    }

    public function getPriceDisk(){
        return $this->priceDisk;
    }

    public function printDisk(){
        echo "Disk         : ". $this->type . " " . $this->capacity. "<br>";
    }
}

?>

File: B/Python/Disk.py
class Disk:
    __type = ""
    __capacity = 0
    __priceDisk = 0
    
    def __init__(self, type, capacity, price):
        self.__type = type
        self.__capacity = capacity
        self.__priceDisk = price
    
    def setType(self, b):
        self.__type = b
    
    def getName(self):
        return self.__type
    
    def setCapacity(self, m):
        self.__capacity = m
    
    def getCapacity(self):
        return self.__capacity
    
    def setPriceDisk(self, m):
        self.__priceDisk = m
    
    def getPriceDisk(self):
        return self.__priceDisk
    
    def printDisk(self):
        print("Disk          : "+str(self.__type))